src/twisted/test/test_tcp.py

```diff
@@ -1190,10 +1194,13 @@ def clientDisconnected(result):
             if not server.lostConnectionReason.check(error.ConnectionClosed):
                 err(server.lostConnectionReason,
                     "Server lost connection for unexpected reason")
-            expectedErrorCode = self.getHandleErrorCode()
+            errorCodeMatcher = self.getHandleErrorCodeMatcher()
```

Comment:
Why is this change necessary?

Reply:
The change in `test_ssl` that adds the nested `contains` matcher is in a test that inherits from _this_ test, which expects the `getHandleErrorCodeMatcher` "abstract method" to return something to compare against the raised exception's first argument.

Now the OpenSSL implementation detail could be one of two messages, and I happen to have introduced the PyHamcrest dependency, so I decided to address that ambiguity with a matcher, and changed the method name (which is in a test and not `proto_helpers`, so a private API) to reflect that.